docs/examples/DataSet/DataSet-class-walkthrough.ipynb

```diff
@@ -0,0 +1,1348 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# DataSet class walkthrough\n",
+    "\n",
+    "In this notebook we are going to go through the mighty `DataSet` class, and get an overview of the most of it's methods and properties. In this case, the more you know, the more you can achieve."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Note on DataSet object creation\n",
+    "\n",
+    "Users are expected to obtain `DataSet` objects in the following ways:\n",
+    "\n",
+    "* run a `Measurement` and get a `DataSet` via a `DataSaver` after exiting the measurement context, as shown here:\n",
+    "\n",
+    "```python\n",
+    "...\n",
+    "meas = Measurement(...)\n",
+    "...\n",
+    "with meas.run() as datasaver:\n",
+    "    ...\n",
+    "dataset = datasaver.dataset  # <- this one\n",
+    "...\n",
+    "```\n",
+    "\n",
+    "* load a `DataSet` using one of the `load_*` functions from `qcodes.dataset.data_set` module, for example, `load_by_run_spec`:\n",
+    "\n",
+    "```python\n",
+    "dataset = load_by_run_spec(\n",
+    "    experiment_name='pinchoff',\n",
+    "    sample_name='SDH-X-13c',\n",
+    "    captured_run_id=12\n",
+    ")\n",
+    "```\n",
+    "\n",
+    "Users should **NOT** instantiate a `DataSet` object via its constructor, or via the `new_data_set` function."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Preparation: a DataSet from a dummy Measurement\n",
+    "\n",
+    "In order to obtain a `DataSet` object, we are going to run a `Measurement` storing some dummy data (see [Dataset Context Manager](Dataset Context Manager.ipynb) notebook for more details)."
```

Comment:
Again here I think the links needs to have spaces escaped.

Reply:
done